### ISSUE TITLE
subread

### ISSUE THREAD

**Issue by chen5027** [ubccr/software-layer#200]
Hello, please also create a module for "subread" so I can run featureCounts.  Thanks!

https://subread.sourceforge.net/featureCounts.html

**Comment by tonykew** [comment]
Test build of Subread version 2.0.4 - more info to follow

**Comment by tonykew** [comment]
Subread version 2.0.4 has been published to the ccrsoft/2023.01 software release


```
vortex1$ module spider subread
----------------------------------------------------------------------------
  subread: subread/2.0.4
----------------------------------------------------------------------------
    Description:
      High performance read alignment, quantification and mutation
      discovery

    Properties:
      Bioinformatic libraries/apps

    You will need to load all module(s) on any one of the lines below before the "subread/2.0.4" module is available to load.

      gcc/11.2.0
[...]
vortex1$ module load gcc/11.2.0 subread/2.0.4
vortex1$ 
```


Subread has several binaries available in the [...]/bin and [...]/bin/utilities
directories, all of which are available in the PATH

```
vortex1$ ls ${EBROOTSUBREAD}/bin
exactSNP       subindel  sublong        subread-buildindex
featureCounts  subjunc   subread-align  utilities
vortex1$ ls ${EBROOTSUBREAD}/bin/utilities
detectionCall  genRandomReads  qualityScores  repair            txUnique
flattenGTF     propmapped      removeDup      subread-fullscan
vortex1$ 
```


For example:

```
vortex1$ subjunc

Version 2.0.4

Usage:

 ./subjunc [options] -i <index_name> -r <input> -o <output>
[...]
vortex1$ 

```
